### ISSUE TITLE
OC Client for openSUSE Tumbleweed

### ISSUE THREAD

**Issue by Wikinaut** [owncloud/client#3768]
I  refer to page https://software.opensuse.org/download/package?project=isv:ownCloud:desktop&package=owncloud-client .

On this page several openSUSE client versions are available, but ownCloud client package for openSUSE TumbleWeed (the current rolling release version) is missing there.

Because I do not know where to correctly file a request for "Owncloud Client for openSUSE Tumbleweed" I file this as an issue here. Perhaps someone can help?

**Comment by dragotin** [comment]
You should be able to use the one from openSUSE Factory.

**Comment by Wikinaut** [comment]
@dragotin You are right, and I use the OC client from Factory since years.

With the issue report I was not clear enough.

I wanted to point out that the present "best" OC client location for openSUSE Tumbleweed users is indeed "Factory" (not 13.2, for example).

But as openSUSE Tumbleweed is for tested programs (like the OC client release), whereas Factory is the bleeding edge of openSUSE, the **Tumbleweed** repo would be the correct place for both the TW and the Factory users - not the Factory repo.


So in short:
Perhaps try to "move" OC client from openSUSE Factory repo to openSUSE Tumbleweed repo.

Just as a remark, there will be soon this "42 LEAP" version of openSUSE.

**Comment by jnweiger** [comment]
https://build.opensuse.org/project/add_repository_from_default_list/isv:ownCloud:desktop shows that we have enabled 'openSUSE Tumbleweed'. It apparently still shows up as Factory everywhere else.

**Comment by Wikinaut** [comment]
@jnweiger danke soweit!
